Editor/MobiledgeXEditorWindow.cs

```diff
@@ -98,7 +98,7 @@ private void Awake()
           }
           private void OnGUI()
           {
-              AssetDatabase.Refresh();
+              
```

Comment:
ok, this as you mentioned, stops blasting Cpu to save every GUI update call.

Reply:
its added (After AddMobiledgeXPlugins )